en/applifters-codex.md

```diff
@@ -28,6 +28,10 @@ If you have a conflict with another Applifter, do not hold it inside. First, try
 
  Only with honesty and sincerity will we be able to keep Applifting a place where it is a joy to be working and where we can talk openly.
 
+### I can accept feedback
```

Comment:
How are you going to bring people higher in EQ if they are not on a level where they can accept feedback?

Reply:
I truly do not know if it's possible. I think that the ability to accept feedback is something which is implicitly part of our culture - and this step only makes it explicit.

Reply:
@kirscfil So you have to hire people who can already accept feedback, right? How do you select for it in the interview process?

Reply:
@petrbazout Eiter me or @Vratislav are present on each interview and assure the cultural compatibility (especially talking about conflicts, feedback and asking about past experience with these).